src/main/groovy/com/vanniktech/android/javadoc/extensions/AndroidJavadocExtension.groovy

```diff
@@ -0,0 +1,37 @@
+package com.vanniktech.android.javadoc.extensions
+
+import org.gradle.api.Project
+
+/**
+ * <p>Created on 23/06/17
```

Comment:
Can you go and remove this alltogether? No file has proper date / authors. You'll still get credited in the release notes for the feature.